tools/run_tests/xds_k8s_test_driver/framework/infrastructure/traffic_director.py

```diff
@@ -290,6 +293,33 @@ def delete_forwarding_rule(self, force=False):
         self.compute.delete_forwarding_rule(name)
         self.forwarding_rule = None
 
+    def create_firewall_rule(self,
+                             force=False,
+                             allowed_ports: List[str] = ['8080-8100']):
```

Comment:
[Using a mutable default value as an argument](https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html) is an anti-pattern. I think it's fair to not provide the default value here.

Reply:
Default value removed.